AppInspector/rules/default/networkcomms/outbound_network.json

```diff
@@ -381,7 +381,7 @@
     "severity": "moderate",
     "patterns": [
       {
-        "pattern": "NetworkInterfaceType\\.Wman|NetworkInterfaceType\\Wwanpp|NetworkInterfaceType\\.Wwanpp2",
+        "pattern": "NetworkInterfaceType\\.Wman|NetworkInterfaceType\\.Wwanpp|NetworkInterfaceType\\.Wwanpp2",
```

Comment:
Might be more readable as `NetworkInterfaceType\\.(Wman|Wwanpp|Wwanpp2)`.

(Also, these are documented here: https://docs.microsoft.com/en-us/dotnet/api/system.net.networkinformation.networkinterfacetype?view=net-5.0 - might want to add to _comments if it isn't there.)